app/League-Entries/page.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { LeagueEntries } from '@/components/LeagueEntries/LeagueEntries';
+import { LeagueSurvivors } from '@/components/LeagueSurvivors/LeagueSurvivors';
+import * as React from 'react';
+
+/**
+ * @returns Leagues
```

Comment:
How did you get by without adding a description of the function? I might have to add a rule if it's not requiring a description.

Reply:
The return is incorrect. It should be @returns { JSX.Element } - Description

Reply:
> How did you get by without adding a description of the function? I might have to add a rule if it's not requiring a description.

I'm not sure, I have no warning or error in my console looking for a function description currently.

![CleanShot 2024-06-18Code001217@2x](https://github.com/LetsGetTechnical/gridiron-survivor/assets/40150036/23ae77b6-fe03-4910-85af-f35e70ecfe44)

Reply:
> The return is incorrect. It should be @returns { JSX.Element } - Description

@shashilo Updated in latest commit.